lib-injection/build/docker/python/dd-lib-python-init-test-django-gunicorn/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.11
```

Comment:
No need for this one, it's a tested app. I mean, maybe python team will want to test python3.11, but it's not related with this current change.

Reply:
and same for any other dockerfile

Reply:
Maybe except `runner`? And `interfaces/schemas`?